index.html

```diff
@@ -1,527 +1,729 @@
-                     <div class="action">
-                        <button>
-                            <span>
-                                See Project
-                            </span>
-                        </button>
-                     </div>
-                </div>
+                  </li>
+                </ul>
+              </div>
             </div>
-            <div class="card-works">
-                <div class="snapshoot-portfolio">
-                    <img src="assets/img/snapshoot-protfolio-2.png" alt="Snapshoot Photo">
-                </div>
-                <div class="left-block">
-                    <div class="primary-txt">
-                        <div class="project-title">
-                            <h2>Multi-Post Stories</h2>
-                        </div>
-                        <div class="client-frame">
-                            <ul>
-                                <li>
-                                    <div class="client-name">
-                                    <h4>CANOPY</h4>
-                                    </div>
-                                </li>
-                                <li>
-                                    <div class="role-txt">
-                                        <span>
-                                            <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                                <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
-                                            </svg>
-                                        </span>
-                                        <h4>Back End Dev</h4>
-                                    </div>
-                                </li>
-                                <li>
-                                    <div class="year-txt">
-                                        <span>
-                                            <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                                <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
-                                            </svg>
-                                        </span>
-                                        <h4>2015</h4>
-                                    </div>
-                                </li>
-                            </ul>
-                        </div>
+            <div class="detail-txt">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="tags">
+              <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <div class="action">
+              <button>
+                See Project
+              </button>
+            </div>
+          </div>
```

Comment:
- [ ] You should stick to the design as much as possible, kindly add more space for the text part and reduce the image size, so your design looks like the Figma one

|Your design|Figma design|
|-|-|
|![Screenshot_20220708_053113](https://user-images.githubusercontent.com/25227022/177917729-2c99c138-a8b3-40e6-ba0f-27de4bf5a892.png)|![Screenshot_20220708_053622](https://user-images.githubusercontent.com/25227022/177917785-93823345-ed62-4f5c-8a47-48b0bd1a8019.png)|